kitsune/users/models.py

```diff
@@ -81,6 +82,7 @@ class Profile(ModelBase, SearchMixin):
     fxa_uid = models.CharField(blank=True, null=True, unique=True, max_length=128)
     fxa_avatar = models.URLField(max_length=512, blank=True, default='')
     products = models.ManyToManyField(Product, related_name='subscribed_users')
+    password_change_time = models.DateTimeField(blank=True, null=True)
```

Comment:
probably we need a better naming for this. As is, it can be confused with the password of the local profile and not from FxA.